libraries/libfc/src/crypto/elliptic_webauthn.cpp

```diff
@@ -88,6 +88,8 @@ struct webauthn_json_handler : public rapidjson::BaseReaderHandler<rapidjson::UT
             return true;
          case IN_NESTED_CONTAINER:
             return true;
+         default:
+            FC_THROW_EXCEPTION( exception, "String: current_state (${current_statea}) out of bound", ("current_state", current_state) );
```

Comment:
I don't think we should make any of these changes to `elliptic_webauthn.cpp`. I'm concerned there is a consensus difference here.

Reply:
We can also silence the warning with `__builtin_unreachable` at the end of the function, I believe. As-is, if it makes it to the end of the function it's pretty much UB anyways :shrug:

Reply:
Thanks. I will do an `elog` and then call `__builtin_unreachable` at the end of the function.